### ISSUE TITLE
WAR Merit "Aggressive Aim" should be adding +25 Ranged Accuracy on initial upgrade.

### ISSUE THREAD

**Issue by Anonyy89** [EdenServer/community#2865]
### Checklist

- [X ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [X ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [X ] I have provided my client version in the details. (type /ver into your game window)

### Details
(Version 30181205_0)

I believe WAR Merit "Aggressive Aim" should be adding +25 Ranged Accuracy on initial upgrade. At the moment it is only adding +4 R.ACC (Which is what it should be adding for subsequent upgrades) on the initial upgrade. 

I tested this on my WAR today, put one point into "Aggressive Aim", did a checkparam, used it, and did another checkparam and the results showed my R.ACC only increased by 4 instead of 25.

Also on a semi unrelated note: I also believe Warcry should be adding R.ATK just like berserk does.

Thanks.

**Comment by Volkaru** [comment]
I'm unable to find any concrete data on Aggressive Aim's initial accuracy. It's probably assumed that, like a lot of group 2 merits, that it's base amount is the same as each incremental upgrade.

Closing until someone can provide additional data suggesting a base amount for Aggressive Aim.

As far as Warcry goes. I found no data suggesting it ever gave a ranged attack bonus. Even current retail it doesn't give R.Atk.
Wiki and BG link in-era of the move: https://www.bg-wiki.com/index.php?title=Warcry&oldid=29757
https://ffxiclopedia.fandom.com/wiki/Warcry?oldid=358846
